src/zenml/io/fileio.py

```diff
@@ -46,27 +46,17 @@ def copy(src: PathType, dst: PathType, overwrite: bool = False) -> None:
     if src_fs is dst_fs:
         src_fs.copy(src, dst, overwrite=overwrite)
     else:
-        if not overwrite and file_exists(str(dst)):
-            raise OSError(
-                (
-                    "Destination file %r already exists and argument `overwrite` is "
-                    "false."
-                )
-                % dst
+        if not overwrite and file_exists(dst):
+            raise FileExistsError(
+                f"Destination file '{convert_to_str(dst)}' already exists "
+                f"and `overwrite` is false."
             )
         contents = open(src, mode="rb").read()
         open(dst, mode="wb").write(contents)
 
 
-def file_exists(path: Union[bytes, str]) -> bool:
-    """Returns true if file exists at path.
-
-    Args:
-        path: Local path in filesystem.
-
-    Returns:
-        True if file exists, else False.
-    """
+def file_exists(path: PathType) -> bool:
+    """Returns `True` if the given path exists."""
```

Comment:
You would take out args and returns in the docstrings?

Reply:
For such super small ones I'd say yes, seems like it's kind of self-explanatory. But can also add it back in if you think that makes sense